core/src/main/java/org/apache/iceberg/view/BaseMetastoreViewCatalog.java

```diff
@@ -161,7 +162,7 @@ private View create(ViewOperations ops) {
               .defaultNamespace(defaultNamespace)
               .defaultCatalog(defaultCatalog)
               .timestampMillis(System.currentTimeMillis())
-              .putSummary("operation", "create")
+              .putAllSummary(EnvironmentContext.get())
```

Comment:
in https://iceberg.apache.org/view-spec/#summary we mention that the summary might contain the engine name/version, so I figured it makes sense to add this info from `EnvironmentContext` here